config/routes.rb

```diff
@@ -318,6 +318,7 @@
     get '/redirect/help_center' => 'redirect/help_center#show', as: :help_center_redirect
     get '/redirect/contact/' => 'redirect/contact#show', as: :contact_redirect
     get '/redirect/policy/' => 'redirect/policy#show', as: :policy_redirect
+    get '/completions_cancel/' => 'completions_cancellation#show'
```

Comment:
Other similar routes are named with `/cancel/` as a standalone path fragment, as a suffix to the base route name (e.g. `/verify/cancel/`, `/sign_up/cancel/`, `/account_reset/cancel/`). I think we should name this similarly for consistency.

```suggestion
    get '/sign_up/completed/cancel/' => 'completions_cancellation#show'
```